frontend/app/src/scenes/Home/scenes/Category/scenes/subcategories/scenes/styles.css

```diff
@@ -0,0 +1,7 @@
+:local .content {
```

Comment:
no need to add `:local` prefix. Every CSS class is `:local` by default.

Reply:
ok